### DISCUSSION TITLE
Vercel Deployment Failed

### DISCUSSION THREAD

**Discussion by Xyliaze** [vercel/community#6391] [Help]
### Summary

I pushed changed to my repository which vercel automatically pulls from. Usually this is all fine. But for some reason all my commits today have failed before vercel pulls them.

There's no indication of the deployment on verbal and the show details button does nothing on GitHub.

![image](https://github.com/vercel/community/assets/144526680/01acd6a0-e76a-4272-977b-6aab5320a6e1)
![image](https://github.com/vercel/community/assets/144526680/29f13ede-9519-4a81-b5f0-7869810ceacb)

### Steps to Reproduce

Push a deployment?

**Comment by amyegan** [comment]
When you click the "Details" link on the "All checks have failed" message, do you see more information about the error?

You can also try a local build to get more details about what went wrong. Make sure you have the latest version of [Vercel CLI](https://vercel.com/docs/cli#updating-vercel-cli) installed and then run [`vercel deploy`](https://vercel.com/docs/cli/deploy#) from your local repo. That will attempt to build the project and log any errors that occur.
